### ISSUE TITLE
Validate allows wildcards

### ISSUE THREAD

**Issue by lko-di** [omichelsen/compare-versions#69]
Hello,

first of all, thanks for this library.

I'm not sure if it is a bug or intended behavior, but `validate` also returns `true` if the given version contains a wildcard (x, *). I'm aware that this lib is already lenient and not strictly adhering to semver, which is fine. However the docs for `validate` don't mention that wildcards are allowed in versions.

```typescript
import { validate } from 'compare-versions';

describe('versions-compare:validate', () => {
  it.each([
    '1.0',
    '1.0.0',
    '1.0.1',
    '1.1.0',
    '1.0.0-rc1',
    '1.0.0-rc.1',
    '1.0.0-rc.1.2',
    '1.0.0-rc.1.2+1',
    '1.0.0-rc.1.2+1.2',
  ])('valid version: %s', (validVersion) => {
    expect(validate(validVersion)).toEqual(true);
  });

  it.each(['x.0.1', '1.x', '1.*', '1.0.x', '1.0.*'])('invalid version: %s', (invalidVersion) => {
    expect(validate(invalidVersion)).toEqual(false);
  });
});
```

I would expect each case of 'invalid version' to fail.

If the current behavior is intended, would it be possible to introduce a new validate function for strict semver or is this out of the scope of this library?

**Comment by omichelsen** [comment]
I can see the argument that `1.x` is a _range_ and not a version (therefore not valid).

Can you explain a little more on your use case?

**Comment by omichelsen** [comment]
I've released v6.1.0 with a new function `validateStrict` which applies the official strict semver rules.
